components/lib/Tooltip/Tooltip.module.scss

```diff
@@ -4,21 +4,51 @@
     color: white;
     padding: 0.35rem 0.45rem;
     font-size: 0.8rem;
-    top: 50%;
-    right: -0.5rem;
-    transform: translate(100%, -50%);
     border-radius: 0.3rem;
     white-space: nowrap;
 
     &::before {
         content: ' ';
         position: absolute;
-        top: 50%;
-        left: 0;
-        transform: translate(-50%, -50%) rotate(45deg);
         z-index: -1;
         border: 0.2rem solid black;
         border-right-color: transparent;
         border-top-color: transparent;
     }
-}
\ No newline at end of file
+}
+
+.right {
```

Comment:
<img width="490" alt="Screenshot 2023-08-13 at 12 28 22" src="https://github.com/Juratbek/upper_frontend/assets/55176773/593c5aa3-ec8f-49ea-bafb-3d63b89467b1">

tooltip is under the main section

Reply:
Had to remove overflow rules for sidebar.